docs/pages/demo/daterangepicker/MinMaxDateRangePicker.example.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import addWeeks from 'date-fns/addWeeks';
+import { Dayjs } from 'dayjs';
+import { Moment } from 'moment';
+import { DateTime } from 'luxon';
+import { makeJSDateObject } from '../../../utils/helpers';
```

Comment:
It won't work in people's codebases.

Reply:
I know, but only one thing we can do without dramatically changes in examples - add warning comment

Reply:
Actually, it won't be the only issue. Doesn't `DateRangePicker` has a required context that we should problem?

I think that we will need to have a dedicated story for this. It's important all the demos can be copied and paste without any exceptions. It hasn't always been the case in the core repository, it has been painful for developers.

Reply:
I mean, this is require to interop with different libraries. 
Ideally we need separate demo for each lib (maybe somehow generated as we are doing for formats)